content/concepts/transport.md

```diff
@@ -115,3 +115,189 @@ and some places in the codebase still use the "swarm" terminology.
+ensure that small malicious packets like SYN packets cannot be sent and consume excessive 
+resources, as can be done with TCP.
+
+QUIC saves another round-trip in using TLS 1.3 by optimistically providing keyshares. 
+If you have established a connection before, the host can send you a session ticket 
+that can be used to establish a new secure connection instantly, without any round-trips.
+
+> Over the last several years, the IETF has been working on a new version of TLS, TLS 1.3.
+> Learn more about TLS 1.3 and how it is used in libp2p on the secure communication concept guide.
+
+<!-- to add link to secure comm guide, and later to the specific doc that covers TLS -->
+
+libp2p only supports bidirectional streams and uses TLS 1.3 by default (but can use other
+cryptography methods). The streams in libp2p map cleanly to QUIC packets.
+
+When a connection starts, peers will take their host key and create a self-signed CA 
+certificate. They then sign an intermediate chain using their self-signed CA and put it 
+as a certificate chain in the TLS handshake.
+
+At the end of the handshake, each peer knows the certificate of the other. The peer can 
+verify if the connection was established with the correct peer by looking up the first 
+CA certificate on the certificate chain, retreive the public key, and using it to calculate 
+the opposing peer ID. QUIC acts like a record layer with TLS 1.3  as the backend as TLS is 
+responsible for all the cryptography.
+
+{{% notice "info" %}}
+
+To be clear, there is no additional security handshake and stream muxer need as QUIC provides 
+all of this by default.
+
+{{% /notice %}}
+
+Following the multiaddress format described earlier, a standard QUIC connection will
+look like:
+
+```
+/ip4/127.0.0.1/udp/65432/quic/
+```
+
+In this section, we offered an overview of QUIC and how QUIC works in libp2p.
+
+{{% notice "tip" %}}
+
+For more details on QUIC, including its limitations 
+check out the following resources:
+
+{{% /notice %}}
+
+## WebTransport
+
+Another transport protocol under development at the IETF is WebTransport.
+WebTransport is a new specification that uses QUIC to offer an alternative to
+WebSockets. Instead, it can be considered WebSockets over QUIC by allowing 
+browsers to establish a stream-multiplexed and bidirectional 
+connection to servers. 
+
+The specification can depend on and reuse the QUIC infrastructure in place 
+to offer WebSockets that take the benefits of UDP and offer sockets without head-of-line 
+blocking.
+
+Recall that WebSockets are bidirectional, full-duplex communication between two 
+points over a single-socket connection. WebTransports, as a result, can be used
+like WebSockets, but with the support of multistreams.
+
+WebTransport streams can be arbitrary in size and independent when possible. 
+They are reliable but can be canceled when possible. The datagrams in a 
+WebTransport connections are MTU-sized and can be unreliable when possible.
```

Comment:
I don't follow what "when possible" means. 

We don't support unreliable datagrams with webtransport

Reply:
Addressed